core/ui/SideBar/Tags.tid

```diff
@@ -0,0 +1,6 @@
+created: 20170204162223518
+list: $:/core/ui/SideBar/sitetitle $:/core/ui/SideBar/sitesubtitle $:/core/ui/SideBar/pagecontrols $:/core/ui/SideBar/search $:/core/ui/SideBar/lists
+modified: 20170206014025058
+title: $:/tags/SideBarSection
```

Comment:
@Marxsal: The name (and location?) of this tid file does not seem right.

Also, all these tiddlers should not have / do not need any of these fields:

modified, created, type => please remove